internal/controllers/machineset/machineset_delete_policy_test.go

```diff
@@ -383,6 +383,10 @@ func TestMachineOldestDelete(t *testing.T) {
 	empty := &clusterv1.Machine{
 		Status: clusterv1.MachineStatus{NodeRef: nodeRef},
 	}
+	mustDeleteMachine := &clusterv1.Machine{
```

Comment:
might be
```suggestion
	deletingMachine := &clusterv1.Machine{
```

Reply:
I called it `mustDeleteMachine` as it's already named that way in `TestMachineToDelete`.